katacomb/scripts/continuum_pipeline.py

```diff
@@ -133,6 +137,39 @@ def create_parser():
     return parser
 
 
+def _infer_defaults_from_katdal(katds):
+    """
+    Infer some default parameters for MFImage and UVBlAvg based
+    upon what we know from the katdal object.
+    """
+
+    uvblavg_params = {}
+    mfimage_params = {}
+
+    # Try and always average down to ~1024 channels if necessary
+    num_chans = len(katds.channels)
+    factor = num_chans // 1024
+    if factor > 1:
+        uvblavg_params['avgFreq'] = 1
+        uvblavg_params['chAvg'] = factor
+
+    # Get the reference antenna used by cal
+    # and use the same one for self-cal
+    ts = katds.source.telstate
+    refant = ts.get('cal_refant')
+    if refant is not None:
```

Comment:
What happens if refant is 'None', will this get dealt with in some later part of the pipeline ?

Reply:
The default MFImage refant will kick in (which is just the antenna with the most unflagged data).